### DISCUSSION TITLE
customer-returns - informacja o zaakceptowaniu zwrotu

### DISCUSSION THREAD

**Discussion by mikolaj-bronk** [allegro/allegro-api#6489] [Suggestions - rejected]
Dzień dobry,

Na podstawie zgłoszenia w https://github.com/allegro/allegro-api/issues/6487 chciałbym przedstawić propozycję rozszerzenia funkcjonalności endpointu "customer-returns" zwracającego informacje o zwrotach klientów o dodatkową informację, która odpowie na pytanie czy dany zwrot został zaakceptowany przez sprzedawcę. 

Sprzedawca ma możliwość zaakceptowania lub odrzucenia zwrotu.
Obecnie nie ma różnicy między statusem domyślnym, z jakim wpada nowe zgłoszenie zwrotu, a statusem potwierdzenia akceptacji takiego zgłoszenia.  Jedyna informacja pojawia się dopiero, gdy sprzedawca odrzuci takie zgłoszenie (w węźle "rejection").

Niestety gdy chcielibyśmy stworzyć zestawienie zwrotów do rozliczeń, to nie możemy brać tych danych, gdyż dopiero po 14 dniach mamy pewność, że taki zwrot nie został odrzucony.

**Comment by MartaNowaczyk** [comment]
@mikolaj-bronk  Sprawdzamy możliwość wdrożenia zwrotu statusu, jak w wątku #5669. Czy taka opcja rozwiązałaby Twój problem?

**Comment by MartaNowaczyk** [comment]
Sugestię oznaczyłam jako odrzuconą, bo dokładnie takiego rozwiązania nie wdrożymy. Chcielibyśmy jednak zwracać wspomniane we wcześniejszym wątku statusy zwrotu, jednak nie mamy jeszcze terminu realizacji. Gdy taka opcja się pojawi na pewno damy znać w oddzielnym komunikacie.
